src/lib/db/indexedDBService.js

```
@@ -206,197 +343,235 @@ async function getHistoryById(id) {
+  return new Promise(async (resolve, reject) => {
+    try {
+      await removeTagFromAllSummaries(id)
+      const transaction = db.transaction([TAGS_STORE_NAME], 'readwrite')
+      const objectStore = transaction.objectStore(TAGS_STORE_NAME)
+      const request = objectStore.delete(id)
+      request.onsuccess = () => resolve(true)
+      request.onerror = (event) => reject(event.target.error)
+    } catch (error) {
+      reject(error)
+    }
+  })
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)